tradeit-android-sdk/src/main/java/it/trade/android/sdk/model/TradeItOrderDetailsParcelable.java

```diff
@@ -3,9 +3,36 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import it.trade.model.reponse.OrderDetails;
+import it.trade.model.reponse.Warning;
+
+public class TradeItOrderDetailsParcelable implements Parcelable {
+    public String orderSymbol;
```

Comment:
@guillaumedebavelaere I updated to move `warnings` to the `orderDetails`. What's our strategy for accessors? Half the classes seem to define them for other fields but then others don't and just mark the field as public.

Reply:
If the user is supposed to modify these fields you can let them public. Otherwise, you can mark them private (or package-private/protected if you need to set them in unit tests) and add a getter.